### DISCUSSION TITLE
The bot used flask_sqlalchemy to error

### DISCUSSION THREAD

**Discussion by flllkis** [eternnoir/pyTelegramBotAPI#1785] [Q&A]
The bot using flask_sqlalchemy will report an error the next day

CR.CR_SERVER_LOST, "Lost connection to MySQL server during query"

pymysql.err.OperationalError: (2013, 'Lost connection to MySQL server during query')

You must restart the bot every day

**Comment by Badiboy** [comment]
Not a library issue.

**Reply by flllkis** [reply]
> 不是图书馆问题。

Is it my code ？ 
I've already used the code
apihelper.SESSION_TIME_TO_LIVE = 5 * 60

**Comment by coder2020official** [comment]
The error itself is telling you the problem. MySQL server is failing at some point
